Fix:  Updater signer failed signing file without extension (#14713)
tauri-apps/tauri: .changes/updater-signer-files-without-extension.md

---

> tauri-cli: patch:bug

Reviewer: ```suggestion
tauri-cli: patch:bug
"@tauri-apps/cli": patch:bug
```

---

> `ext.to_os_string().unwrap()` returns None on files without an extension in `sign_file()`

Reviewer: ```suggestion
`tauri signer sign` doesn't work for files without an extension
```
